app/policies/contact_log_recipient_policy.rb

```diff
@@ -1,5 +1,2 @@
 class ContactLogRecipientPolicy < ContactLogPolicy
-  def index?
-    false
```

Comment:
this was causing the contact log system usage items to not show up, I handle this by hiding the index button in the UI instead